nixos/modules/services/x11/desktop-managers/gnome.nix

```diff
@@ -357,6 +357,8 @@ in
         isSystem = true;
       };
 
+      services.orca.enable = mkDefault (notExcluded pkgs.orca);
```

Comment:
`notExcluded` already contains `mkDefault` since it signifies optionality.

Reply:
Ugh. It's defined differently in different modules. Fixed GNOME, the other two do need it.